.pre-commit-config.yaml

```diff
@@ -65,3 +65,11 @@ repos:
         language: golang
         require_serial: true
         pass_filenames: false
+  - repo: local
```

Comment:
Happy to change this to use a repo like https://github.com/Bahjat/pre-commit-golang or https://github.com/dnephin/pre-commit-golang